.version.yml

```diff
@@ -5,4 +5,4 @@
 # e.g. site audits, automated backups.
 version: 8
 type: paas
-scaffold: 1.1.2
+scaffold: 2.0.0
```

Comment:
Bumped major version as this is a breaking change scaffold (e.g you cannot upgrade from `1.x` without manual solr upgrade)